shared/types/customTypes.ts

```diff
@@ -60,6 +60,9 @@ export enum RoleType {
     Permission.VersionRead |
     Permission.RoadmapReadUsers,
   Business = Permission.TaskRead |
+    Permission.TaskCreate |
```

Comment:
This would need a corresponding migration to update the existing business users in the database.

Reply:
A good catch

Reply:
Migration added. I suppose I Cant directly use RoleType.Business in the migration as future changes to that type would screw up the old migrations, so I just have 
```

const oldBusinessRole =
  Permission.TaskRead |
  Permission.TaskEdit |
  Permission.TaskDelete |
  Permission.TaskRate |
  Permission.TaskRatingEdit |
  Permission.TaskValueRate |
  Permission.CustomerRepresent |
  Permission.RoadmapReadUsers;

const newBusinessRole = oldBusinessRole | Permission.TaskCreate;

```